collections/_article/godot-game-awards-stayathomedev.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Meet the Creator Behind the Godot Game Awards"
+excerpt: "StayAtHomeDev has been sharing his passion in YouTube for over two years and now he's running the awards."
+categories: ["showcase"]
+author: Emi
+image: /storage/blog/covers/godot-awards-stayathomedev.webp
+date: 2024-12-09 15:00:00
+---
+
+If you’ve watched Godot-related content on YouTube, [StayAtHomeDev](https://www.youtube.com/@stayathomedev) needs no introduction. He’s been sharing his passion for game development and the Godot Engine through his channel. As the creator of the the popular “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, and the Godot Game Awards, he’s helped showcase the amazing work of the Godot community. We had the chance to talk with him about his journey, what inspired the awards, and his efforts to support the community.
+
+### What inspired you to create the Godot Game Awards, and how long has it been running?
+I started the “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series over two years ago on my channel.  At first it started as a compilation of cool projects or games I found throughout each week.  About 7-8 videos in, I thought it would be cool if people could vote for their favorite each week and the engagement exploded.  People would comment every week and eventually I compiled every winner into a final vote for the “[Best Godot Project of the Year](https://www.youtube.com/watch?v=3_3N_nL9gIM)” in 2023.  This year I wanted to expand that into multiple categories, highlighting certain achievements like art design and music, and give attention to other things like plugins, tutorials, even devlogs.  From that came the Godot Game Awards, a compilation of the best Godot-based games, plugins, and projects as voted by viewers on my channel.
```

Comment:
```suggestion
I started the “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series over two years ago on my channel.  At first it started as a compilation of cool projects or games I found throughout each week.  About 7-8 videos in, I thought it would be cool if people could vote for their favorite each week and the engagement exploded.  People would comment every week and eventually I compiled every winner into a final vote for the “[Best Godot Project of the Year](https://www.youtube.com/watch?v=3_3N_nL9gIM)” in 2023.

This year I wanted to expand that into multiple categories, highlighting certain achievements like art design and music, and give attention to other things like plugins, tutorials, even devlogs.  From that came the Godot Game Awards, a compilation of the best Godot-based games, plugins, and projects as voted by viewers on my channel.
```